src/command/cmd_funcs.c

```diff
@@ -10835,3 +10835,15 @@ cmd_vcard_save(ProfWin* window, const char* const command, gchar** args)
     cons_show("User vCard uploaded");
     return TRUE;
 }
+
+gboolean
+cmd_executable_async(ProfWin* window, const char* const command, gchar** args)
+{
+    if (args[1] == NULL) {
+        cons_bad_cmd_usage(command);
```

Comment:
> Thank you for your contribution. Took a glance, with just 1 edit (return after bad usage) it will look suitable for PR for me. Seems like you copied way to run executables from editor.c.

I did took the code from editor.c :wink: